### PR TITLE
updated fonts and font sizes to be synced with design system and dial…

### PR DESCRIPTION
**Pull request by rachana-egov**
…og size

### CODE REVIEW

File: packages/digit_components/lib/widgets/digit_dialog.dart [hunk 1]
```diff
@@ -36,24 +37,28 @@ class DigitDialog extends StatelessWidget {
...
           if (options.enableRecordPast == true)
             Padding(
-              padding: const EdgeInsets.all(8.0),
+              padding: const EdgeInsets.all(0),
               child: Row(
                 children: <Widget>[
                   if (options.secondaryAction != null)
                     SizedBox(
-                      width: MediaQuery.of(context).size.width / 3,
+                      width: 50,
```
Comment (naveen-egov) [inline packages/digit_components/lib/widgets/digit_dialog.dart:61]: why is it hardcoded to 50

File: packages/digit_components/lib/widgets/digit_dialog.dart [hunk 2]
```diff
@@ -89,9 +94,7 @@ class DigitDialog extends StatelessWidget {
               children: <Widget>[
                 if (options.primaryAction != null)
                   Padding(
-                    padding: options.dialogPadding != null
-                        ? options.dialogPadding!
-                        : const EdgeInsets.all(0),
+                    padding: const EdgeInsets.all(0),
```
Comment (naveen-egov) [inline packages/digit_components/lib/widgets/digit_dialog.dart:97]: If we don't have the padding remove the padding instead of adding 0 padding

**Review by naveen-egov — CHANGES_REQUESTED** [review]

File: packages/digit_components/lib/widgets/digit_dialog.dart (hunk 1 above)
Comment (rachana-egov) [reply packages/digit_components/lib/widgets/digit_dialog.dart]: fixed

File: packages/digit_components/lib/widgets/digit_dialog.dart (hunk 2 above)
Comment (rachana-egov) [reply packages/digit_components/lib/widgets/digit_dialog.dart]: fixed

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**